pages/ksphere/konvoy/1.4/upgrade/upgrade-kubernetes-addons/index.md

```diff
@@ -154,10 +154,10 @@ kind: ClusterConfiguration
 apiVersion: konvoy.mesosphere.io/v1alpha1
 spec:
   kubernetes:
-    version: 1.16.9
+    version: 1.16.12
   containerNetworking:
     calico:
-      version: v3.13.3
+      version: v3.13.4
   addons:
     configRepository: https://github.com/mesosphere/kubernetes-base-addons
```

Comment:
```suggestion
  - configRepository: https://github.com/mesosphere/kubernetes-base-addons
```

Reply:
Already fixed in https://github.com/mesosphere/dcos-docs-site/pull/3062/files